src/Workspaces/Core/Portable/Rename/RenameLocations.cs

```diff
@@ -123,6 +123,7 @@ private class SearchResult
         /// <summary>
         /// Find the locations that need to be renamed.
         /// </summary>
+        /// <remarks>Do not call this method directly.  Instead, use <see cref="Renamer.FindRenameLocationsAsync"/></remarks>
```

Comment:
Why?

Reply:
because that will be the location that does the dispatching from local to oop.  but acutally, i'm going to swap that around.

Reply:
undone.